model/form-entities-table.js

```diff
@@ -269,12 +269,31 @@ module.exports = memoize(function (db) {
 				isEntitiesNestedMap = entityObjects.value instanceof this.database.NestedMap;
 				if (!isEntitiesNestedMap) return result;
 				Item = entityObjects.value.getItemType();
+				// FIXME: This is probably broken. foreach loop over dataForms isn't correct.
+				// FIXME: it should be iteration over dataForm.map. Look under toWSSchema method.
 				Item.prototype.getBySKeyPath(this.sectionProperty).forEach(function (section) {
 					result = result.concat(section.toMetaDataJSON());
 				});
 
 				return result;
 			}
+		},
+		toWSSchema: {
+			value: function (ignore) {
+				var entityObjects, Item, isEntitiesNestedMap, schema = {};
+				schema[this.propertyName] = {
+					type: "array",
+					items: { type: "object", properties: {} }
+				};
+				entityObjects = this.propertyMaster.resolveSKeyPath(this.propertyName);
+				isEntitiesNestedMap = entityObjects.value instanceof this.database.NestedMap;
+				if (!isEntitiesNestedMap) return schema;
+				Item = entityObjects.value.getItemType();
+				Item.prototype.getBySKeyPath(this.sectionProperty).map.forEach(function (section) {
+					Object.assign(schema[this.propertyName].items.properties, section.toWSSchema());
+				}.bind(this));
```

Comment:
Just a side note, `forEach` will auto bind its second param, so you could write simply:

`...forEach(function (...) {..}, this)`

withoud explicit `blind`